KIF Tests/CollectionViewTests.m

```diff
@@ -59,6 +59,27 @@ - (void)testTappingLastItemAndSectionWithoutAnimation
 }
 
 
+- (void)testTappingLastAndFirstRowAtPositionWithAccessiblityIdentifier
```

Comment:
I would rename this and the other test to `testScrolling` since you're not really tapping anything